crates/uv-platform-tags/src/platform_tag.rs

```diff
@@ -50,19 +56,19 @@ pub enum PlatformTag {
     /// Ex) `android_21_x86_64`
     Android { api_level: u16, arch: Arch },
     /// Ex) `freebsd_12_x86_64`
-    FreeBsd { release: String, arch: Arch },
+    FreeBsd { release_arch: SmallString },
```

Comment:
It's simpler to just store this as `"12_x86_64"` rather than `"12"` and `Arch::X86_64`. We don't use the structured values anywhere (these variants are rare).